src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.ts

```diff
@@ -196,22 +229,17 @@ export const createDashboard = async (
         ...(sameType ? panelToUpdate.explicitInput : {}),
 
         ...incomingEmbeddable.input,
-        id: incomingEmbeddable.embeddableId as string,
+        id: incomingEmbeddable.embeddableId,
 
         // maintain hide panel titles setting.
         hidePanelTitles: panelToUpdate.explicitInput.hidePanelTitles,
       };
     } else {
       // otherwise this incoming embeddable is brand new and can be added via the default method after the dashboard container is created.
-      untilDashboardReady().then(async (container) => {
-        container.addNewEmbeddable(incomingEmbeddable.type, incomingEmbeddable.input);
-      });
+      untilDashboardReady().then((container) =>
+        container.addNewEmbeddable(incomingEmbeddable.type, incomingEmbeddable.input)
+      );
     }
-
-    untilDashboardReady().then(async (container) => {
-      container.setScrollToPanelId(incomingEmbeddable.embeddableId);
-      container.setHighlightPanelId(incomingEmbeddable.embeddableId);
-    });
```

Comment:
Why was the scrolling behavior removed here for incoming embeddables? Maybe got lost in a merge conflict?

Reply:
Ah true, good catch! While fixing this, I also noticed that the code didn't scroll to a newly added panel, only one that was edited, so I added some extra logic to make it scroll to newly added panels.

Reply:
Fixed in 6120d3fbe7a